.github/workflows/main.yml

```diff
@@ -85,6 +85,7 @@ jobs:
       - run: cargo test -F $FEATURE -F profiler
       - run: cargo test --workspace -F $FEATURE --tests -- --ignored --test-threads 1
       - run: cargo test -F $FEATURE -F disable-dev-mode -p risc0-r0vm
+      - run: cargo test -F $FEATURE -F enable-fault-proof -p risc0-zkvm -- tests::memory_io tests::memory_access
```

Comment:
This will re-run all the other tests, so I think we should just add `-F enable-fault-proof` to the normal invocation of `cargo test -p risc0-zkvm`

Reply:
This actually only runs the two tests. My local output:
```
% cargo test -F enable-fault-proof -p risc0-zkvm -- tests::memory_io tests::memory_access 
   Compiling risc0-zkvm v0.18.0 (/Users/erik/risc0/risc0/zkvm)
   Compiling risc0-zkvm-methods v0.18.0 (/Users/erik/risc0/risc0/zkvm/methods)
risc0-zkvm-methods-guest: Starting build for riscv32im-risc0-zkvm-elf), risc0-zkvm-methods(build)                            
risc0-zkvm-methods-guest:     Finished release [optimized + debuginfo] target(s) in 0.16suild)                               
risc0-zkvm-methods-std: Starting build for riscv32im-risc0-zkvm-elf-zkvm-methods(build)                                      
risc0-zkvm-methods-std:     Finished release [optimized] target(s) in 0.09s
    Finished test [optimized + debuginfo] target(s) in 18.75s
     Running unittests src/lib.rs (target/debug/deps/risc0_zkvm-758a526f4cedc86e)

running 2 tests
test host::server::exec::tests::memory_access ... ok
test host::server::prove::tests::memory_io ... ok

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 91 filtered out; finished in 51.33s

   Doc-tests risc0-zkvm

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 16 filtered out; finished in 0.02s
```

Reply:
Thanks for the clarification!